cli/command/container/attach.go

```diff
@@ -103,10 +102,7 @@ func runAttach(dockerCli command.Cli, opts *attachOptions) error {
 	}
 
 	resp, errAttach := client.ContainerAttach(ctx, opts.container, options)
-	if errAttach != nil && errAttach != httputil.ErrPersistEOF {
```

Comment:
The `httputil.ErrPersistEOF` is no longer used, so should no longer be returned by anything.

That said; the cli could connect to an older daemon that still returns this error, so I think we should change these to an `//nolint`, which is what we did in the engine;

```
	//nolint:staticcheck // ignore SA1019 for connecting to old (pre go1.8) daemons
```

https://github.com/moby/moby/commit/fd65fed81bdc5b50b89c3c3ec4f733562564e190 (updated in https://github.com/moby/moby/commit/5f47cef514acba3d0fa0856064057d4c7f218c31)

Reply:
Oh, actually; so that moby change is the exact path we're hitting here in the `ContainerAttach`, so the client won't return a `httputil.ErrPersistEOF` error,  so we can safely remove that condition here.